openage/convert/entity_object/export/formats/nyan_file.py

```diff
@@ -61,11 +60,11 @@ def dump(self):
             output_str += "import "
 
             for part in fqon:
-                output_str += "%s." % (part)
+                output_str += f"{part}."
 
             output_str = output_str[:-1]
 
-            output_str += " as %s\n" % (alias)
+            output_str += f" as {alias}\n"
```

Comment:
Not a flynt problem, but it would have been better to use join here :D Shame on me